src/linear_algebra/basic.lean

```diff
@@ -87,36 +87,38 @@ begin
   { intro i, exact (h i).map_zero },
 end
 
-variable (R)
+variables (R M)
```

Comment:
Can you add `(α : Type*)` here so that it appears before the typeclass arguments like it did before?

I assume the intent of this change is to make `α` an explicit variable?

Reply:
Done.